rpa-bot/1.6/modules/ROOT/pages/configuration.adoc

```diff
@@ -156,6 +157,74 @@ A confirmation pop-up displays to indicate the setting was fixed successfully. I
 
 To fix the terminal server setting, which is needed for Secure Sessions, ensure the software is installed. Visit xref:hardware-software-requirements.adoc[Hardware and Software Requirements] for more information about requirements for Secure Sessions.
 
+[[diagnostic-infos]]
+== Diagnostic Infos
+
+The diagnostics panel shows important runtime information to help you identify current and potential problems with RPA Bot. The panel also shows which Windows session is currently being used by which secure session. Some values cannot be determined when RPA Bot is not running.
+
+image:bot-configurator-diagnostic-infos.png[The RPA Bot Configurator application showing the Diagnostic Infos panel, 50%, 50%]
+
+* *Bot Service state*
++
+Shows the current status of the Windows service.
+* *Bot start time (local time)*
++
+Shows the last bot start time and running time. The time is shown in the time zone of the current RPA Bot user.
+* *Process execution count*
++
+Shows the number of processes that executed successfully since the last bot start time.
+* *Workflow execution count*
++
+Shows the number of workflows executed successfully since the last bot start time.
+* *Average Secure session count*
++
+Shows the average number of secure sessions at the same time, the bot needs to run the processes since it was last started.
+* *Max. Secure session count*
++
+Shows the maximum number of secure sessions at the same time the bot needs to run the processes since it was last started.
```

Comment:
```suggestion
Shows the maximum number of secure sessions at the same time the bot needs to run the processes since it was last started.
Could needs be must?
```